src/main/java/core/basesyntax/Info.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Info {

```

Comment:
Interface names could be adjectives and end with “able” or “ible” whenever the interface provides a capability; otherwise, they should be nouns.
Here I'd recommend naming your interfaces 'Drawable'